src/components/AnnouncementsPage/Announcement.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(theme => ({
+	subject: {
+		color: theme.palette.primary.main,
+		fontWeight: theme.typography.fontWeightBold,
+		paddingTop: theme.spacing(4)
+	},
+	timestamp: {
+		color: theme.palette.secondary.light
+	},
+	body: {
+		paddingBottom: theme.spacing(4)
+	}
+}));
+
+function Announcement({ subject, timestamp, body }) {
+	const classes = useStyles();
+
+	return (
+		<Container>
+			<Typography variant='h4' component='h2' className={classes.subject}>
+				{subject}
+			</Typography>
+			<Typography variant='h6' component='h3' className={classes.timestamp}>
+                Posted on: {timestamp}
```

Comment:
Let's do the timestamp stuff here.
So normally the timestamp data will be passed in like some sort of string: `"2021-01-21T08:13:11Z"`

In JavaScript, we can use the built-in `Date` object.
```js
let date = new Date("2021-01-21T08:13:11Z");
date.getDate();
...
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date 

That way we should be able to format the string into the format that we want.